### PR TITLE
README: add new web client

### PR DESCRIPTION
**Pull request by kosmgco**
added a link to the web client at https://tldr.ooops.me

### CODE REVIEW

**Comment by ivanhercaz** [comment]
Seems nice, beautiful, simple and... has multilingual support for tldr-pages! I really like it, @kosmgco. Congrats for your work. Is it available on GitHub?

**Comment by schneiderl** [comment]
Looks great 👍

**Comment by ivanhercaz** [comment]
> Is there anything specific in this client, something that makes it special? Maybe it could be noted after the link, since it's hard to tell what `tldr.oops.me` is (unlike `tldr.js` and `DistroWatch`).

I think the most remarkable feature is that tldr.oops.me supports multilingual pages, an unavailable feature in `tldr.js` and DistroWatch. So the special thing might be "a tldr-pages web client with multilingual support".

**Comment by kosmgco** [comment]
> Is there anything specific in this client, something that makes it special? Maybe it could be noted after the link, since it's hard to tell what `tldr.oops.me` is (unlike `tldr.js` and `DistroWatch`).

1. It will sync tldr pages to local database every day, don't need to request api.github.com per request.
2. It can query by platform or language or both. 
3. When click the `<code>` tag(highlight text), It will copy to clipboard and easy to use in other place.
4. Some hot tags will be displayed, click then the tldr page will show onto the page.

Above of information need to append to README.MD?

**Comment by kosmgco** [comment]
> Seems nice, beautiful, simple and... has multilingual support for tldr-pages! I really like it, @kosmgco. Congrats for your work. Is it available on GitHub?

Thanks. I'm planning to make the project open source on GitHub.

**Comment by zdroid** [comment]
@kosmgco I suggest you add the note about multilingual support. That'd be great.

**Comment by kosmgco** [comment]
@zdroid Sorry to say that I have some trouble with GitHub. I have been contacted GitHub for help. When my repo is restored I will commit it.

Thanks.

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ You can access these pages on your computer using one of the following clients:
 - Web clients:
     - [tldr.jsx](https://github.com/ostera/tldr.jsx): http://tldr.ostera.io/
     - [DistroWatch](https://distrowatch.com/dwres.php?resource=man-pages)
-    - [tldr.ooops.me](https://tldr.ooops.me)
+    - [tldr.ooops.me](https://tldr.ooops.me): a tldr-pages web client with multilingual support
```
Comment (zdroid) [inline README.md]:
```suggestion
   - [tldr.ooops.me](https://tldr.ooops.me): web client with multilingual support
```
it's better this way, thanks for staying with us!

**Pull request merged**